src/redux/actions/fetchEpisodes.js

```diff
@@ -1,12 +1,7 @@
-// export function fetchEpisodes() {
-//   return (dispatch) => {
-//     dispatch({ type: '' })
-//   }
-// }
-
 export const fetchEpisodes = () => dispatch => {
```

Comment:
Same approach as for reducers should be for actions. File name I can suggest is 'episodesActions' or 'episodes'. [Here](https://daveceddia.com/where-fetch-data-redux/) is an example of such a file and how to fetch data in action.

Also [actions doc](https://redux.js.org/advanced/asyncactions)